src/resources/LibraryStyles.css

```diff
@@ -429,27 +429,12 @@ button {
 	opacity: 1;
 }
 
-.SearchBar button,
 .SearchBar button {
-	border: none;
+	border: 0;
 	outline: none;
 	height: 100%;
 	background-color: transparent;
 	transition: 0.15s;
-}
-
-.SearchBar button {
-	color: #ddd;
-	transition: 0.15s;
-}
-
-.SearchBar button {
```

Comment:
this is not used?

Reply:
@QilongTang the `.SearchBar button` selector was repeated several times so I combined all styles applied. Those I've left there are the used ones

Reply:
by checking the code I can see that the value
`color: #ddd;`
is set several times to different values, then if you combined all the styles for this button, how did you know which is the right color?

Reply:
@RobertGlobant20  If load the component using the browser, opening devtools you will see the styles which are actually in use.
![Image 30-10-23 at 12 50](https://github.com/DynamoDS/librarie.js/assets/111511512/61487705-a2f1-4601-bce7-3bb334dfd184)